web/src/components/task/task-sidebar/task-sidebar.tsx

```diff
@@ -512,70 +512,72 @@ const TaskSidebar: FC<TaskSidebarProps> = ({ jobSettings, viewMode, isNextTaskPr
                             {isValidation && !splitValidation && (
```

Comment:
What do you think about moving all this wrapper together with the buttons to a separated component?

Reply:
I was thinking about this but decided to leave as is for now - it's + - okay at this stage